<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="settings_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="get_attack_key">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack Key</string>
     </property>
    </widget>
    <widget class="QPushButton" name="get_place_key">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Place Key</string>
     </property>
    </widget>
    <widget class="QSlider" name="cps_place">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="cps_attack">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>20</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="humanize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Humanize</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="toggle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle</string>
     </property>
    </widget>
    <widget class="QLabel" name="place_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Place CPS (0)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="attack_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack CPS (0)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="config_group">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>9</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>mousePressEvent(QMouseEvent*)</signal>
 </slots>
</ui>
